theano/gpuarray/dnn.py

```diff
@@ -1428,21 +1428,18 @@ def make_node(self, dy, sm):
 
 
 @register_opt2([AbstractConv2d, AbstractConv2d_gradWeights,
-                AbstractConv2d_gradInputs], 'conv_dnn', 'cudnn', 'gpuarray', 'fast_compile')
+                AbstractConv2d_gradInputs], 'fast_compile')
 def local_abstractconv_cudnn_graph(op, context_name, inputs, outputs):
     if (not isinstance(op, (AbstractConv2d,
                             AbstractConv2d_gradWeights,
                             AbstractConv2d_gradInputs))):
-        return None
-
-    inp1 = inputs[0]
-    inp2 = inputs[1]
+        return
 
-    if (node.op.filter_dilation != (1, 1)):
+    if (op.filter_dilation != (1, 1)):
         return None
 
-    if not isinstance(inp1.type, GpuArrayType):
-        return None
+    inp1 = as_gpuarray_variable(inputs[0], context_name)
```

Comment:
I think you're viewing commit by commit. This has been reverted. Earlier, i suspected that the input variable is not on the GPU, hence hasn't been moved.